site/en/blog/soft-navigations-experiment/index.md

```diff
@@ -0,0 +1,258 @@
+
+The reality is always a little trickier than the ideal, and the popular Single Page Application architecture has [never been fully supported by the Core Web Vitals metrics](https://web.dev/vitals-spa-faq/). Rather than loading distinct, individual web pages as the user navigates about the site, these web applications use so-called “soft navigations”, where the page content is instead changed by JavaScript. In these applications, the illusion of a traditional webpage architecture is maintained by altering the URL and pushing previous URLs in the browser’s history to allow the back and forward buttons to work as the user would expect.
+
+Many JavaScript frameworks use this model, but each in a different way. Since this is outside of what the browser traditionally understands as a “page”, measuring this has always been difficult: where is the line to be drawn between an interaction on the _current_ page, versus considering this as a _new_ page?
+
+The Chrome team has been been considering this challenge for some time now, and is looking to standardize a definition of what is a “soft-navigation”, and how the Core Web Vitals can be measured for this—in a similar way that websites implemented in the traditional multi-page architecture (MPA) is currently measured. While still in early stages, the team is now ready to make our work so far more widely available through the use of an origin trial to allow sites that wish to trial this to provide feedback on the approach so far.
+
+## What is a soft navigation?
+
+We have come up with the following definition of a _soft navigation_:
+
+- The navigation is initiated by a user action.
+- The navigation results in a visible URL change to the user, and a history change.
+- The navigation results in a DOM change.
+
+For some sites, these heuristics may lead to false positives (that users would not really consider a "navigation" to have happened) or false negatives (where the user does consider a “navigation” to have happened despite not missing the above criteria). We welcome feedback at [the soft navigation specification repository](https://github.com/WICG/soft-navigations/issues) on the heuristics.
+
+### How does Chrome implement soft navigations?
+
+Once the soft navigation heuristics are enabled (more on this in the next section), Chrome will change the way it reports some performance metrics:
+
+- A `soft-navigation` [`PerformanceTiming`](https://developer.mozilla.org/docs/Web/API/PerformanceTiming) event will be emitted after each soft navigation is detected, with a corresponding `soft-navigation` entry.
+- The performance API will provide access to a `soft-navigation` event timing entry, as emitted by the above `PerformanceTiming` event.
+- The [First Paint](https://developer.mozilla.org/docs/Glossary/First_paint), [First Contentful Paint (FCP), Largest Contentful Paint (LCP)](https://web.dev/fcp/) metrics will be reset, and re-emitted on the next appropriate occurrences of these.
```

Comment:
```suggestion
- The [First Paint (FP)](https://developer.mozilla.org/docs/Glossary/First_paint), [First Contentful Paint (FCP), Largest Contentful Paint (LCP)](https://web.dev/fcp/) metrics will be reset, and re-emitted on the next appropriate occurrences of these.
```

Reply:
Other metrics have abbreviations after them (and you refer to this as FP later).